### DISCUSSION TITLE
data for next day is missing

### DISCUSSION THREAD

**Discussion by morph027** [mampfes/ha_epex_spot#114] [General]
Hm, somehow the spot data for the next day disappeared....it only shows until midnight, even after 14:00 when new data should be present.

It was working in the past.

Tested awattar and smaard. Am i doing something wrong?

![2024-03-04_09-32](https://github.com/mampfes/ha_epex_spot/assets/600106/4c7613a0-5374-42dd-80d9-1383f9edb506)
![2024-03-04_09-33](https://github.com/mampfes/ha_epex_spot/assets/600106/097378aa-607d-4409-a298-7c322636289e)

**Comment by mampfes** [comment]
The screenshot looks good, or? There is no data because it is before 14:00.

**Reply by morph027** [reply]
Ah, yes, but it stays like this, even after new data should have been polled. Will add another screenshot later.

**Comment by mampfes** [comment]
The SmartEnergy API is not providing data for tomorrow at the time of writing (16:50). Please reach out to them if you are using SmartEnergy.

**Comment by morph027** [comment]
Ah, looks better now.

![image](https://github.com/mampfes/ha_epex_spot/assets/600106/6262f304-948c-41cd-99f2-b0fc066d0034)
